pages/docs/manual/latest/installation.mdx

```diff
@@ -13,24 +13,36 @@ canonical: "/docs/manual/latest/installation"
 
 ## New Project
 
+### Quick Setup
+
+```sh
+npm create rescript-app
```

Comment:
npm or `npx create-rescript-app`?

Reply:
why not everything?
<img width="727" alt="image" src="https://github.com/rescript-association/rescript-lang.org/assets/18074327/17f568e2-00a1-45f1-aadd-66cb28c37847">

Reply:
hahaha